### PR TITLE
fix(virtualRepeat): Add ability for container to resize after creation

### PR DESCRIPTION
**Pull request by kseamon**
Step 1 of addressing #4314 and #4336

### CODE REVIEW

**Comment by kseamon** [comment]
@jelbourn

File: src/components/virtualRepeat/virtual-repeater.js [hunk 1]
```diff
@@ -126,17 +126,30 @@ function VirtualRepeatContainerController($$rAF, $parse, $scope, $element, $attr
...
+//
+//   // TODO: Come up with a more robust (But hopefully also quick!) way of
+//   // detecting that we're not visible.
+//   if ($attrs.ngHide) {
+//     $scope.$watch($attrs.ngHide, angular.bind(this, function(hidden) {
+//       if (!hidden) {
+//         $$rAF(angular.bind(this, this.updateSize));
+//       }
+//     }));
+//   }
```
Comment (jelbourn) [inline src/components/virtualRepeat/virtual-repeater.js]: Remove commented-out code and just leave the TODO

File: src/components/virtualRepeat/virtual-repeater.js [hunk 2]
```diff
@@ -126,17 +126,30 @@ function VirtualRepeatContainerController($$rAF, $parse, $scope, $element, $attr
...
+
+  var boundUpdateSize = angular.bind(this, this.updateSize);
```
Comment (jelbourn) [inline src/components/virtualRepeat/virtual-repeater.js]: Could use a comment to explain what's going on here.

File: src/components/virtualRepeat/virtual-repeater.js [hunk 3]
```diff
@@ -126,17 +126,30 @@ function VirtualRepeatContainerController($$rAF, $parse, $scope, $element, $attr
...
+  $$rAF(function() {
+    boundUpdateSize();
+
+    var jWindow = angular.element($window);
+    jWindow.on('resize', boundUpdateSize);
```
Comment (jelbourn) [inline src/components/virtualRepeat/virtual-repeater.js]: This should probably be debounced

File: src/components/virtualRepeat/virtual-repeater.js (hunk 1 above)
Comment (kseamon) [inline src/components/virtualRepeat/virtual-repeater.js]: Meant to remove that and forgot. TODO is no longer relevant.

**Pull request closed without merging**